src/uct/base/uct_md.h

```diff
@@ -30,10 +30,11 @@
 
 
 #define UCT_MD_MEM_DEREG_CHECK_PARAMS(_params, _invalidate_supported) \
-    if (!UCT_MD_MEM_DEREG_FIELD_VALUE(_params, memh, FIELD_MEMH, NULL)) { \
+    if (ENABLE_PARAMS_CHECK && \
```

Comment:
here we still need ENABLE_PARAMS_CHECK because there is access to it

Reply:
removed param check macro